src/components/molecules/Settings/ProjectList/ProjectCell/index.tsx

```diff
@@ -29,47 +31,77 @@ export type Props = {
 const ProjectCell: React.FC<Props> = ({ project, onSelect }) => {
   const intl = useIntl();
   const theme = useTheme();
+  const [isHover, setHover] = useState(false);
 
   return (
-    <Wrapper project={project} onClick={() => onSelect?.(project)}>
-      <Title size="xl" color={theme.projectCell.text}>
-        {project.name ? project.name : intl.formatMessage({ defaultMessage: "No Title Project" })}
-      </Title>
-      <Desc size="s" color={theme.projectCell.text}>
-        {project.description
-          ? project.description
-          : intl.formatMessage({ defaultMessage: "No Description..." })}
-      </Desc>
-      <PublicationStatus status={project.status} />
-    </Wrapper>
+    <StyledWrapper project={project}>
+      <Wrapper
+        project={project}
+        onClick={() => onSelect?.(project)}
+        isHover={isHover}
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}>
+        <Title size="l" color={theme.projectCell.title}>
+          {project.name ? project.name : intl.formatMessage({ defaultMessage: "No Title Project" })}
+        </Title>
+        {isHover && (
+          <DescriptionWrapper>
+            <Desc size="s" color={theme.projectCell.description} isParagraph={true}>
+              {project.description
+                ? project.description
+                : intl.formatMessage({ defaultMessage: "No Description..." })}
+            </Desc>
+          </DescriptionWrapper>
+        )}
+        <Public status={project.status} />
+      </Wrapper>
+    </StyledWrapper>
   );
 };
 
-const Wrapper = styled.div<{ project: Project }>`
+const StyledWrapper = styled.div<{ project: Project }>`
   background: ${props =>
-    props.project.imageUrl ? `url(${props.project.imageUrl})` : props.theme.colors.bg[4]};
-  background-size: cover;
-  color: ${props => props.theme.projectCell.text};
-  padding: 10px;
+    props.project.imageUrl ? `url(${props.project.imageUrl})` : `url(${defaultProjectImage})`};
+  background-size: ${props => (props.project.imageUrl ? "cover" : "400px 240px")};
+  background-position: center;
   box-shadow: 0 0 5px ${props => props.theme.projectCell.shadow};
-  margin: 10px 0;
-  padding: 28px;
-  // position: relative;
-  width: 100%;
-  height: 223px;
+  height: 240px;
+`;
+
+const Wrapper = styled.div<{ project: Project; isHover?: boolean }>`
```

Comment:
The project prop isn't being used. It is used in the StyledWrapper above which it looks like you split into two styled components. Remove here and above in the return